tools/cravatp_score_and_annotate/cravatp_submit.xml

```diff
@@ -0,0 +1,302 @@
+          <option value="Kidney-Papillary-Cell">Kidney-Papillary-Cell</option>
+          <option value="Liver-Nonviral">Liver-Nonviral</option>
+          <option value="Liver-Viral">Liver-Viral</option>
+          <option value="Lung-Adenocarcinoma">Lung-Adenocarcinoma</option>
+          <option value="Lung-Squamous-Cell">Lung-Squamous-Cell</option>
+          <option value="Other">Other</option>
+          <option value="Ovary">Ovary</option>
+          <option value="Pancreas">Pancreas</option>
+          <option value="Prostate-Adenocarcinoma">Prostate-Adenocarcinoma</option>
+          <option value="Rectum">Rectum</option>
+          <option value="Skin">Skin</option>
+          <option value="Stomach">Stomach</option>
+          <option value="Thyroid">Thyroid</option>
+          <option value="Uterus">Uterus</option>
+        </param>
+      </when>
+      <when value="CHASM+VEST">
+        <param format="tabular" name="classifier" type="select" label="CHASM Classifier">
+          <option value="Bladder">Bladder</option>
+          <option value="Blood-Lymphocyte">Blood-Lymphocyte</option>
+          <option value="Blood-Myeloid">Blood-Myeloid</option>
+          <option value="Brain-Glioblastoma-Multiforme">Brain-Glioblastoma-Multiforme</option>
+          <option value="Brain-Lower-Grade-Glioma">Brain-Lower-Grade-Glioma</option>
+          <option value="Breast">Breast</option>
+          <option value="Cervix">Cervix</option>
+          <option value="Colon">Colon</option>
+          <option value="GID">GID</option>
+          <option value="Head-and-Neck">Head-and-Neck</option>
+          <option value="Kidney-Chromophobe">Kidney-Chromophobe</option>
+          <option value="Kidney-Clear-Cell">Kidney-Clear-Cell</option>
+          <option value="Kidney-Papillary-Cell">Kidney-Papillary-Cell</option>
+          <option value="Liver-Nonviral">Liver-Nonviral</option>
+          <option value="Liver-Viral">Liver-Viral</option>
+          <option value="Lung-Adenocarcinoma">Lung-Adenocarcinoma</option>
+          <option value="Lung-Squamous-Cell">Lung-Squamous-Cell</option>
+          <option value="Other">Other</option>
+          <option value="Ovary">Ovary</option>
+          <option value="Pancreas">Pancreas</option>
+          <option value="Prostate-Adenocarcinoma">Prostate-Adenocarcinoma</option>
+          <option value="Rectum">Rectum</option>
+          <option value="Skin">Skin</option>
+          <option value="Stomach">Stomach</option>
+          <option value="Thyroid">Thyroid</option>
+          <option value="Uterus">Uterus</option>
+        </param>
+      </when>
+    </conditional>
+    <param format="tabular" name="GRCh" type="select" label="Genome Reference Consortium Human Build (GRCh)"
+        help="The default human reference genome used for annotation is GRCh38, released on December 24th, 2013 from the Genome Reference Consortium.">
+      <option value="GRCh38">GRCh38/hg38</option>
```

Comment:
just for you information, we could in theory retrieve this information from the dbkey of a dataset, but we will leave it as it is for now.

Reply:
That's a good point. I'll comment this in as a to-do for the future.